src/Beacon.jsx

```diff
@@ -157,6 +164,23 @@ export class Beacon extends Component {
         this.setState({ targetElement, targetBounds }); // eslint-disable-line react/no-did-update-set-state
       }
     }
+
+    if (this.state.tooltip && !this.state.arrowPosition) {
+      // Calculate relative arrow position when ref is available
+      setTimeout(() => {
+        const arrowStyleLeft = this.getArrowHorizontalPosition(
+          targetElement.getBoundingClientRect(),
+          this.refs['tooltip'].getBoundingClientRect(),
```

Comment:
I've just read this in the official docs, so used that format instead as it seemed more friendly to things we don't even use

> If you want to preserve Google Closure Compiler advanced-mode crushing resilience, make sure to never access as a property what was specified as a string. This means you must access using this.refs['myRefString'] if your ref was defined as ref="myRefString".